src/all.ts

```diff
@@ -109,7 +107,6 @@ export {
     Gaia,
     MouseBehaviour,
     KeyboardBehaviour,
-    ChannelType,
```

Comment:
In std.ts gibts zB auch einen ChannelType. Ich nehme an den braucht man dort dann auch nicht?
Bitte kontrollieren für alle Types die rausfliegen.

Reply:
ist dort auch schon entfernt worden 👍